docs/doxygen/pages.dox

```diff
@@ -188,6 +188,9 @@ Some configuration settings are C pre-processor constants, and some are function
 @section MQTT_PINGRESP_TIMEOUT_MS
 @copydoc MQTT_PINGRESP_TIMEOUT_MS
 
+@section MQTT_RECV_POLLING_TIMEOUT_MS
+@copydoc MQTT_RECV_POLLING_TIMEOUT_MS
```

Comment:
In the doc for `MQTT_Init`, we state the following:
```
If there is no time implementation, it is the responsibility of the application to provide a dummy function to always return 0
```
We need to update this to also that `MQTT_RECV_POLLING_TIMEOUT_MS` must also be set to 0 to prevent an infinite loop.

However, this is now insufficient as one iteration of `MQTT_ReceiveLoop` or `MQTT_ProcessLoop` may not be enough to receive the entire payload. The way around that is to set `MQTT_RECV_POLLING_TIMEOUT_MS` to a desired number of iterations, then pass a timer query function that increments on each call. However, that contradicts labeling the unit of time as milliseconds. However, why does the timeout need to be in milliseconds here anyway?

Reply:
I think that setting the timer function to a value that returns 0 can also result in an infinite loop for the `MQTT_ProcessLoop` function as this is the condition used for determining that the timeout period has expired:
```
if( elapsedTimeMs > timeoutMs )
{
                break;
}
```
We should change that condition to be `>=` instead...

Reply:
The goal is to allow unbound number of retries for partial reads of packet data over the network as long as the transport recv function is able to successfully read data over the network. (Reason is that the `recvExact` function is called _after_ some part of the MQTT packet is already read, and thus, we want to attempt to read the remaining packet data as long as the transport recv function does not return an error).
Thus, even if a dummy timer function that always returns 0 is used, as long as the transport receive function is able to perform _partial_ reads, the transport recv calls will be retried.

If a dummy timer that always returns 0 is used and the network read returns 0 bytes, there still wouldn't be an infinite loop inside the `recvExact` function as it uses the `>=` check for the timeout.

Reply:
> We should change that condition to be >= instead...
That definitely looks like a typo as it is now. MQTT_ReceiveLoop uses `>=`, which looks correct

Reply:
https://github.com/FreeRTOS/coreMQTT/pull/120/commits/32f029190780689881d7de7db6053a9b01e438ff
Have fixed it for `MQTT_ProcessLoop` in the commit